cerebrium/getting-started/introduction.mdx

```diff
@@ -21,12 +21,54 @@ Setting up and deploying an app on Cerebrium takes just a few steps:
 
 ### 1. Install the CLI
 
+<Tabs>
+<Tab title="Python (pip)">
+
 ```bash
 pip install cerebrium
+```
+
+</Tab>
+<Tab title="macOS (Homebrew)">
```

Comment:
Could we put this first, I'd prefer not to push people towards `pip` if possible - I doubt people will normally click away from the first tab and I suspect most users are on macs 🤔